memory/recent.py

```
@@ -17,19 +17,44 @@
+        """确保指定角色的历史记录已加载到内存中（懒加载）"""
+        if lanlan_name in self.user_histories:
+            return
+        if lanlan_name in self.log_file_path and os.path.exists(self.log_file_path[lanlan_name]):
+            self.user_histories[lanlan_name] = self._load_history_from_file(
+                self.log_file_path[lanlan_name], lanlan_name,
+            )
+        else:
+            self.user_histories[lanlan_name] = []
+
+    def _ensure_path_for(self, lanlan_name):
+        """确保指定角色有对应的文件路径，对未知角色创建默认路径"""
```

Ruff (0.15.5)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)